src/components/Display.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <input className="display" type="text" value={result} />
+);
+
+Display.propTypes = {
+  result: PropTypes.node.isRequired,
```

Comment:
- [ ] Also the 'result' should be a prop of the type string, not a node. Kindly update the proptype to string.